recipes/libsrtp/all/conanfile.py

```diff
@@ -54,12 +54,18 @@ def source(self):
     def generate(self):
         tc = CMakeToolchain(self)
         tc.variables["ENABLE_OPENSSL"] = self.options.with_openssl
-        tc.variables["TEST_APPS"] = False
+        if  Version(self.version) < "2.6.0":
+            tc.variables["TEST_APPS"] = False
+        else:
+            tc.variables["LIBSRTP_TEST_APPS"] = False
         if Version(self.version) < "2.4.0":
             # Relocatable shared libs on Macos
             tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0042"] = "NEW"
-        if Version(self.version) >= "2.5.0":
+        if  "2.5.0" <= Version(self.version) < "2.6.0":
             tc.cache_variables["BUILD_WITH_WARNINGS"] = False
+        if  "2.6.0" <= Version(self.version):
+            tc.cache_variables["ENABLE_WARNINGS"] = False
+            tc.cache_variables["ENABLE_WARNINGS_AS_ERRORS"] = False
```

Comment:
```suggestion
            tc.cache_variables["ENABLE_WARNINGS_AS_ERRORS"] = False
        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```
The oldest libsrtp version sets CMake policy version to 2.8. Need to set CMP0077 for tc.variables.